README.adoc

```diff
@@ -516,6 +516,24 @@ external_services:
     url: VALUE
 ----
 
+|`JAEGER_SERVICE`
+|The name of the Jaeger service.(default is `jaeger-query`)
```

Comment:
I'd add some comment to clarify when/where those are used.
i.e. Kiali uses JAEGER_SERVICE to connect internally (within the cluster) to Jaeger service to query XYZ.
JAEGER_URL is used to connect Kiali UI to Jaeger service outside the cluster.

Or something like that to make it fully clear when one or another would be needed.
It is correct as it is now, but it lacks some context.

Same comment for Grafana ones.